src/main/java/org/sonar/plugins/clojure/sensors/leinNvd/LeinNvdParser.java

```diff
@@ -0,0 +1,43 @@
+package org.sonar.plugins.clojure.sensors.leinNvd;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import org.sonar.api.utils.log.Logger;
+import org.sonar.api.utils.log.Loggers;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LeinNvdParser {
+    private static final Logger LOG = Loggers.get(LeinNvdParser.class);
+
```

Comment:
Add a private constructor since this is a utility class and we don't want it to be instantiated.

Reply:
ok